activitypub/apmodels/utils.go

```diff
@@ -60,3 +61,50 @@ func Serialize(obj vocab.Type) ([]byte, error) {
 
 	return b, err
 }
+
+// MakeLocalIRIForStreamURL will return a full IRI for the local server stream url.
+func MakeLocalIRIForStreamURL() *url.URL {
+	host := data.GetServerURL()
+	u, err := url.Parse(host)
+	if err != nil {
+		log.Errorln("unable to parse local IRI stream url", err)
+		return nil
+	}
+
+	u.Path = path.Join(u.Path, "/hls/stream.m3u8")
+
+	return u
+}
+
+// MakeLocalIRIforLogo will return a full IRI for the local server logo.
+func MakeLocalIRIforLogo() *url.URL {
+	host := data.GetServerURL()
+	u, err := url.Parse(host)
+	if err != nil {
+		log.Errorln("unable to parse local IRI stream url", err)
+		return nil
+	}
+
+	u.Path = path.Join(u.Path, "/logo/external")
+
+	return u
+}
+
+// GetLogoType will return the rel value for the webfinger repsonse
```

Comment:
Linter is reporting a missing period at the end of the comment.

Reply:
Done.